packages/core/admin/server/src/services/auth.ts

```diff
@@ -88,14 +82,11 @@ const forgotPassword = async ({ email } = {} as { email: string }) => {
 
 /**
  * Reset a user password
- * @param {Object} param params
- * @param {string} param.resetPasswordToken token generated to request a password reset
- * @param {string} param.password new user password
```

Comment:
can we keep the jsdocs , but without the object definitions? I think the comments could be quite useful